source/speech/extensions.py

```diff
@@ -7,10 +7,17 @@
 Extension points for speech.
 """
 
-from extensionPoints import Action
+from extensionPoints import Action, Filter
+from speech.types import SpeechSequence
 
 speechCanceled = Action()
 """
 Notifies when speech is canceled.
 Handlers are called without arguments.
 """
+
+filterSpeechSequence = Filter[SpeechSequence]()
```

Comment:
Convention for extension points:

```suggestion
filter_speechSequence = Filter[SpeechSequence]()
```